src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewTime.razor.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+using System.Globalization;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class ViewTime : ComponentBase
+{
+
+    #region Declarations
+
+    [Parameter]
+    public int? Hour { get; set; }
+
+    [Parameter]
+    public int? Minute { get; set; }
+
+    [Parameter]
+    public int? Second { get; set; }
+
+    [Parameter]
+    public int? Millisecond { get; set; }
+
+    [Parameter]
+    public string Format { get; set; } = "HH:mm:ss";
+
+    [Parameter]
+    public CultureInfo CultureInfo { get; set; } = CultureInfo.CurrentCulture;
+
+    public string DisplayTime
+    {
+
+        get
+        {
+            string ReturnTime = String.Empty;
```

Comment:
```suggestion
            var returnTime = String.Empty;
```
local variables should be camelCase

Reply:
resolved on latest commit